dms/views/dms_file.xml

```diff
@@ -240,7 +240,11 @@
                                         <div
                                             class="o_kanban_record_title o_text_overflow"
                                         >
-                                            <field name="name" />
+                                            <field
+                                                name="content"
+                                                filename="name"
+                                                widget="binary"
+                                            />
```

Comment:
I think it is not necessary because the action of download already exists.
![ejemplo](https://github.com/OCA/dms/assets/4117568/5c4fd192-1c12-44e3-9877-fb4c4ddf4855)

Reply:
I will be honest: I did not see that download button, it's still necessary to make one more click but at least you don't have to open/close the form-view everytime.. so yes, it's indeed less necessary than I thought 👍